webcrypto-shim.js

```diff
@@ -19,7 +19,7 @@ module.exports = function webcryptoShim (global) {
         _SubtleCrypto = global.SubtleCrypto || _subtle.constructor || Object,
         _CryptoKey  = global.CryptoKey || global.Key || Object;
 
-    var isEdge = window.navigator.userAgent.indexOf('Edge/') > -1
+    var isEdge = global.navigator.userAgent.indexOf('Edge/') > -1
```

Comment:
global is a local variable, it is the variable that is assigned to window or self.